tensorboard/webapp/app_routing/store/app_routing_selectors.ts

```diff
@@ -116,11 +116,17 @@ export const getExperimentIdToExperimentAliasMap = createSelector(
   getRouteKind,
   getRouteParams,
   (routeKind, routeParams): {[id: string]: ExperimentAlias} => {
-    if (routeKind !== RouteKind.COMPARE_EXPERIMENT) {
+    const compareParams = routeParams as CompareRouteParams;
+    if (
+      routeKind !== RouteKind.COMPARE_EXPERIMENT &&
+      !(
+        routeKind === RouteKind.CARD &&
+        compareParams.experimentIds.indexOf(',') >= 0
```

Comment:
Does the second condition depends when you call getExperimentIdToExperimentAliasMap? For example, if it's single experiment, do we still call this selector?

Also I think what you mean is -1. IndexOf returns -1 if not found.

Reply:
Sounds good, changed condition to check for -1.